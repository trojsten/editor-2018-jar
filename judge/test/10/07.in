INT:
velkostmapy
16
INT:
dlzka
0
INT:
vyska
0
INT:
okuliare
0
VECTOR_STR:
mapa
16
LF>VT%A=QH1*HASE
;T2>Z36WF34Z&1=A
KSI+JX(QG)GPLKU?
ZF7@:VH>O$O=W,66
P1%32>0BCO4&:+08
XS6X8F?F?%'=MOW9
P:%6&SSS?%6F00E/
[+?TW<&47)@*[/FK
W)//5<A8MU/0&Z2-
Z[;BM=FLCY0,&W,;
-PU(R%X+R0)S(W-T
'5V.#####C9MU79;
U141#U4%#E[%L8UK
=+??#####>=[ARH*
>BGM;UT&L.7,$/XH
+0NH?<L<7BV+/NZU
