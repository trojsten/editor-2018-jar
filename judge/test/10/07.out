{"dlzka": 5, "vyska": 3}